Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs

```diff
@@ -1579,7 +1579,7 @@ private void AddLabel(PropertyItem pi, Grid propertyPanel, ref FrameworkElement
                             if (propertyControl != null)
                             {
                                 propertyPanel.RowDefinitions.Add(new System.Windows.Controls.RowDefinition());
-                                Grid.SetColumnSpan(labelPanel, 2);
+                                Grid.SetColumnSpan(labelPanel, 1);

```

Comment:
isn't 1 default?
then the line can be removed?

Reply:
I think your suggestion is much better than what I proposed initially. I tested your suggestion, and the results remain the same, which is good :)

![image](https://github.com/PropertyTools/PropertyTools/assets/133324134/28ba0f9f-33cb-446f-acdd-991897042de7)

![image](https://github.com/PropertyTools/PropertyTools/assets/133324134/85a737e6-aa17-428f-97de-31f424e005b4)